src/transforms/tree.js

```diff
@@ -217,7 +221,7 @@ function nodePath(node) {
 }
 
 function nodeName(node) {
-  return node.id.split("/").pop();
+  return node.id.replace(/\\\//g, "\0").split("/").pop().replace(/\0/g, "/");
```

Comment:
This makes me realize that we should handle escaped backslashes like we do in d3-hierarchy:

https://github.com/d3/d3-hierarchy/blob/7bea49efdc1093b28a8d60d2f8717d8a17803564/src/stratify.js#L125-L133

We shouldn’t use \0 and hope that it doesn’t exist in the path. 😄